AUTHORS

```diff
@@ -11,3 +11,6 @@ Anand Suresh
 Brett Bergmann
 Jesse Friedman
 Zach Bjornson <zbbjornson@gmail.com>
+
+Greta.io
+Dennis Mårtensson <dennis@greta.io>
```

Comment:
This is the instructions:

```
# Names should be added to this file as:
# Name or Organization <email address>
# The email address is not required for organizations
```
So I read it as adding the org and then the person in that org.